Slim/Formats/Playlists/CUE.pm

```diff
@@ -308,19 +740,22 @@ sub parse {
 		}
 
 		# Merge in file level attributes
-		for my $attribute (qw(CONTENT_TYPE ALBUMARTIST ARTIST ALBUM YEAR GENRE DISC DISCC COMMENT 
-			                  REPLAYGAIN_ALBUM_GAIN REPLAYGAIN_ALBUM_PEAK ARTISTSORT ALBUMARTISTSORT ALBUMSORT COMPILATION)) {
-
-			if (!exists $track->{$attribute} && defined $cuesheet->{$attribute}) {
-					
-				# Bug 18110 - only merge ALBUMARTIST/ARTISTSORT if the track's ALBUMARTIST/ARTIST is the same as the album's
-				next if $attribute =~ /(.*)SORT$/ && $track->{$1} ne $cuesheet->{$1};
-
-				$track->{$attribute} = $cuesheet->{$attribute};
-
-				main::DEBUGLOG && $log->debug("    $attribute: $track->{$attribute}");
-			}
-		}
+                # Removed in order to consider all the attributes at Album level.
+                #
+                # for my $attribute (qw(CONTENT_TYPE ALBUMARTIST ARTIST ALBUM YEAR GENRE DISC DISCC COMMENT 
+		#	                  REPLAYGAIN_ALBUM_GAIN REPLAYGAIN_ALBUM_PEAK ARTISTSORT ALBUMARTISTSORT ALBUMSORT COMPILATION))
+                for my $attribute (keys %$cuesheet){
+            
+                    if (!exists $track->{$attribute} && defined $cuesheet->{$attribute}) {
+
+                        # Bug 18110 - only merge ALBUMARTIST/ARTISTSORT if the track's ALBUMARTIST/ARTIST is the same as the album's
+                        next if $attribute =~ /(.*)SORT$/ && $track->{$1} ne $cuesheet->{$1};
+
+                        $track->{$attribute} = $cuesheet->{$attribute};
+
+                        main::DEBUGLOG && $log->debug("    $attribute: $track->{$attribute}");
+                    }
+                }
 		
```

Comment:
Most of this block seems to be modified only because tabs have been replaced with spaces? Could you please make sure you use tabs, not spaces, to indent text in all of the file? Thanks!